app/assets/stylesheets/theme-selector.scss

```
@@ -153,6 +153,35 @@
       font-size: 0.5em;
     }
   }
+
+  @media screen and (max-width: 810px) {
+    .level_tag {
+      display: none;
+    }
+  }
+}
+
+.level-bar__labels {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  padding: 0.3rem var(--su-5);
+  color: black;
+  border-radius: 20px;
+
+  background: black;
+  background: -moz-linear-gradient(left, #66e2d5 0%, #fefa87 50%, #ff0000 100%);
+  background: -webkit-linear-gradient(
+    left,
+    #66e2d5 0%,
+    #fefa87 50%,
+    #ff0000 100%
+  );
+  background: linear-gradient(to right, #66e2d5 0%, #fefa87 50%, #ff0000 100%);
```

We automatically add vendor prefixes so for development we don't need to add them.

Also, in terms of colors, we're currently turning towards using our native CSS variables instead of hex values OR SCSS variables. [There's a file with all of them](https://github.com/forem/forem/blob/master/app/assets/stylesheets/config/_colors.scss).

So in this case I'd suggest little change:

```suggestion
  background: linear-gradient(to right, var(--accent-success) 0%, var(--accent-warning) 50%, var(--accent-danger) 100%);
```